roles/setup_repo/tasks/PG_RedHat_setuprepos.yml

```diff
@@ -149,12 +151,31 @@
   failed_when: >
     reposub.rc != 0 or 'error: ' in reposub.stdout.lower()
   changed_when: reposub.rc == '0'
-  when: >
-    tpa_subscription_token|length > 0
-    and install_bdr_packages|bool
-    and pg_version|int >= 14
+  when:
+    - enable_edb_repo|bool
+    - tpa_subscription_token|length > 0
+    - install_bdr_packages|bool
+    - pg_version|int == 14
+    - bdr_version|int == 4
   loop: "{{ edb_2q_repositories }}"
 
+- name: Install PGD RPM packages repo
+  ansible.builtin.yum_repository:
+    name: "{{ item.name }}"
+    description: "{{ item.description }}"
+    file: "enterprisedb-postgres_distributed"
+    baseurl: "{{ item.repo_url }}"
+    gpgkey: "{{ edb_pgd_rpm_gpg_key }}"
+    repo_gpgcheck: true
+    enabled: true
+  when:
+    - enable_edb_repo|bool
+    - repo_token|length > 0
+    - install_bdr_packages|bool
+    - pg_version|int >= 15
+    - bdr_version|int >= 5
+  loop: "{{ pgd_rpm_repo }}"
```

Comment:
If the `pgd_rpm_repo` includes both x86_64 and noarch, wouldn't it either install the same repo twice or, if on an aarch64 host, fail when attempting to download the x86_64 repo?